types/src/transaction/script.rs

```diff
@@ -251,15 +251,16 @@ pub struct ScriptFunction {
     module: ModuleId,
     function: Identifier,
     ty_args: Vec<TypeTag>,
-    args: Vec<TransactionArgument>,
+    #[serde(with = "vec_bytes")]
+    args: Vec<Vec<u8>>,
```

Comment:
This doesn't have anything to do with this PR but is there a reason why we have the type info and the args themselves separate instead of doing something like `args: Vec<(TypeTag, Vec<u8>)>`?

Reply:
Ah, that `ty_args` is not the type of the argument, it is used to represent the actual types given to the type parameter, i.e., the actual type of the generics.